telemetry/constants.go

```diff
@@ -0,0 +1,27 @@
+package telemetry
```

Comment:
copyright? Can you check this for all the new files you are adding for AI related PRs?